src/main.css

```diff
@@ -8,6 +8,8 @@
 
 #nav1 {
   height: 40%;
+  padding-left: 10px!important;
```

Comment:
Try to minimize the use of important, this usually denotes the HTML is wrongly setup.